src/components/Add.js

```diff
@@ -0,0 +1,39 @@
+import React,{useState} from 'react';
+
+export default function Add(props) {
+    // Adding a hook to get the input and stores it.
+    const [input, setInput] = useState('');
+    
+    // Function that add items to the list
+    const addItem = () => {
+        if (!input) return null;
+        props.setItems((prevItems) => {
```

Comment:
It is better to destruct here or in the function component.
Done here:
```
const { setItems } = this.props;
setItems((prevItems) => {
```

Done in the function component definition:
```
export default function Add({setItems})
```